ui/app/contexts/metametrics.js

```diff
@@ -18,14 +19,71 @@ import { getEnvironmentType } from '../../../app/scripts/lib/util'
 import {
   sendMetaMetricsEvent,
   sendCountIsTrackable,
+  mixpanelId,
 } from '../helpers/utils/metametrics.util'
 
+const analytics = new Analytics(mixpanelId)
```

Comment:
This is actually a segment ID, right? Not a Mixpanel ID? It looks like this is expecting a Segment "Write Key".

Reply:
ah, my bad. I'll fix this naming

Reply:
This has been renamed, and the environment variable changed in circle.